code/modules/plumbing/plumbers/iv_drip.dm

```diff
@@ -7,11 +7,11 @@
 	density = TRUE
 	use_internal_storage = TRUE
 
-/obj/machinery/iv_drip/plumbing/Initialize(mapload)
+/obj/machinery/iv_drip/plumbing/Initialize(mapload, bolt=anchored, layer)
```

Comment:
```suggestion
/obj/machinery/iv_drip/plumbing/Initialize(mapload, bolt, layer)
```
Setting the param to the anchored var kinda ruins the point of having that parameter